CHANGELOG.md

```diff
@@ -5,6 +5,28 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.03.2-ce (2017-05-29)
+
+### Networking
+
+- Fix a concurrency issue preventing network creation [#33273](https://github.com/moby/moby/pull/33273)
+
+### Runtime
+
+- Relabel secrets path to avoid a Permission Denied on selinux enabled systems [#33236](https://github.com/moby/moby/pull/33236)
++ Add a new mount `z` option to instruct that a local volume need to be relabeled [#33236](https://github.com/moby/moby/pull/33236)
```

Comment:
I haven't looked at the pr yet as I'm on mobile, but this should just be fixing some edge axes around the "z"option. The option itself has been around for awhile.